src/components/Forms/MaskedTextField.js

```diff
@@ -285,6 +290,7 @@ class MaskedTextField extends React.Component {
             name={name}
             aria-required={required}
             aria-invalid={hasError}
+            aria-describedby={hasError ? `hint_${inputId} error_${inputId}` : `hint_${inputId}`}
```

Comment:
Which element gets the `error_<input id>` class? I can't find one at a glance but I remember it possibly being part of the form system?

Reply:
`ValidationError`